### ISSUE TITLE
 Tree table selection is not working for Shift key

### ISSUE THREAD

**Issue by lk-granit** [primefaces/primeng#13690]
### Describe the feature you would like to see added

Could you please take a look into it.
Multiple selection in Tree table with metaKeySelection = "true" is not working for Shift key.
https://primeng.org/treetable#selectionmultiple

**Comment by dalenguyen** [comment]
I'm not sure if on purpose, but it only works for the `Command / Windows` or `Control` key.

**Comment by lk-granit** [comment]
Yes, it's only work Command / Windows or Control key. But for large amount of data selection it's not convenient.

**Comment by autonomobil** [comment]
Any update? There's also 
https://github.com/primefaces/primeng/issues/8035
https://github.com/primefaces/primeng/issues/7998
